config/detekt.yml

```diff
@@ -0,0 +1,381 @@
+autoCorrect: true
+failFast: false
+
+test-pattern: # Configure exclusions for test sources
+  active: true
+  patterns: # Test file regexes
+    - '.*/test/.*'
+    - '.*Test.kt'
+    - '.*Spec.kt'
+  exclude-rule-sets:
+    - 'comments'
+  exclude-rules:
+    - 'NamingRules'
+    - 'WildcardImport'
+    - 'MagicNumber'
+    - 'MaxLineLength'
+    - 'LateinitUsage'
+    - 'StringLiteralDuplication'
+    - 'SpreadOperator'
+    - 'TooManyFunctions'
+
+build:
+  warningThreshold: 0
+  failThreshold: 0
```

Comment:
Nit: I get deprecation warnings for `warningThreshold` and `failThreshold`. Seems we can remove these two lines in favour of maxIssues: 0 which is already there?

Reply:
Interesting. I generated this with the `detektGenerateConfig` gradle command (and then just set them to 0). Well, I'll remove them. :)